privacy/ccpa/policy.go

```diff
@@ -44,7 +55,17 @@ func ReadFromRequestWrapper(req *openrtb_ext.RequestWrapper) (Policy, error) {
 }
 
 func ReadFromRequest(req *openrtb2.BidRequest) (Policy, error) {
-	return ReadFromRequestWrapper(&openrtb_ext.RequestWrapper{BidRequest: req})
+	var gpp gpplib.GppContainer
+	var err error
+	if req.Regs != nil && len(req.Regs.GPP) > 0 {
+		gpp, err = gpplib.Parse(req.Regs.GPP)
+		if err != nil {
+			// zero out bad container
+			gpp = gpplib.GppContainer{}
```

Comment:
This behavior isn't present in the auction.go code. Should it be? .. or would it be good to ensure that gpp is always empty if a Parse fails? Should it be nil? Just looking for consistency with the two call sites.

Reply:
The container as defined there is a struct rather than a pointer, so cannot be nil. It is a valid point about being inconsistent, I think I just wanted to not ignore an error if it appeared. We may still want to empty out the container if the parse fails in the gpp library itself.

Reply:
Gotcha, Works for me at this stage. We can reconsider in Phase 2.